src/utils/string.js

```diff
@@ -29,3 +29,36 @@ export function removeURLs(
   );
   return string.replace(regExp, '$1' + redactedText);
 }
+
+/**
+ * Take an absolute file path string and sanitize it except the last file name segment.
+ *
+ * Note: Do not use this function if the string not only contains a file path but
+ * also contains more text. This function is intended to use only for path strings.
+ */
+export function removeFilePath(
+  filePath: string,
+  redactedText: string = '<PATH>'
+): string {
+  let pathSeparator = null;
+
+  // Figure out which separator the path uses.
+  if (filePath.includes('/')) {
+    // This is a Unix-like path.
+    pathSeparator = '/';
+  } else if (filePath.includes('\\')) {
+    // This is a Windows path.
+    pathSeparator = '\\';
+  }
```

Comment:
nit: maybe you can use `lastIndexOf` already here, so that we know the index right away. It will return -1 if not found.

With the current implementation we have this extra `lastIndexOf` later that is useless because we already looked up the character here. This isn't such a big deal for these small strings but is easy to do.